### PR TITLE
Update the catalan localization with the newly added lines

### PR DESCRIPTION
**Pull request by ivanbea**
Update the catalan localization with the newly added lines

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -603,12 +603,12 @@ msgid "Check your inbox for an email with the confirmation code to enter below:"
 msgstr "Comprova el teu correu per rebre el codi de confirmació i entra'l aquí sota:"
 
 #: src/view/com/modals/Threadgate.tsx:72
-msgid "Choose \"Everybody\" or \"Nobody\""
-msgstr "Tria \"Tothom\" or \"Ningú\""
+msgid "Choose \"Everybody\" or \"Nobody\
+msgstr "Tria \"Tothom\" or \"Ningú\
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:607]: I think there might be some quotation marks missing here

File: src/locale/locales/ca/messages.po [hunk 2]
```diff
@@ -748,7 +748,7 @@ msgstr "Codi de confirmació"
 
 #: src/view/com/modals/Waitlist.tsx:120
 msgid "Confirms signing up {email} to the waitlist"
-msgstr ""
+msgstr "Confirma afegir {enail} a la llista d'espera"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:751]: Small typo here, should be {email}

File: src/locale/locales/ca/messages.po (hunk 1 above)
Comment (ivanbea) [reply src/locale/locales/ca/messages.po]: Thanks! Corrected

File: src/locale/locales/ca/messages.po (hunk 2 above)
Comment (ivanbea) [reply src/locale/locales/ca/messages.po]: Thanks! Corrected

**Comment by ivanbea** [comment]
Added new localizations, you can check it @jordimas @darccio @ferranrego @miqueladell @mvdan @johnnydement @mejans

File: src/locale/locales/ca/messages.po [hunk 3]
```diff
@@ -2189,20 +2188,20 @@ msgstr "Cap descripció"
...
 #: src/view/com/composer/text-input/mobile/Autocomplete.tsx:97
 #: src/view/com/composer/text-input/web/Autocomplete.tsx:191
 msgid "No result"
 msgstr "Cap resultat"
 
 #: src/view/screens/Feeds.tsx:456
-msgid "No results found for \"{query}\""
-msgstr "No s'han trobat resultats per \"{query}\""
+msgid "No results found for \"{query}\
+msgstr "No s'han trobat resultats per \"{query}\
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:2204]: Some missing quotation marks here

File: src/locale/locales/ca/messages.po [hunk 4]
```diff
@@ -2277,72 +2276,72 @@ msgstr "Només {0} poden respondre."
...
+msgstr "Obre els enllaços al navegador de l'aplicació"
 
 #: src/view/com/pager/FeedsTabBarMobile.tsx:81
 msgid "Open navigation"
 msgstr "Obre la navegació"
 
 #: src/view/screens/Settings.tsx:737
 msgid "Open storybook page"
-msgstr ""
+msgstr Obre la pàgina de l'historial
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:2296]: Missing quotation marks around this

File: src/locale/locales/ca/messages.po [hunk 5]
```diff
@@ -2437,23 +2436,23 @@ msgstr "Contrasenya actualitzada!"
...
 
 #: src/Navigation.tsx:153
 msgid "People following @{0}"
-msgstr ""
+msgstr "Oersones seguint a @{0}"
 
 #: src/view/com/lightbox/Lightbox.tsx:66
 msgid "Permission to access camera roll is required."
-msgstr ""
+msgstr "Cal permís per accedir al carret de la càmera. "
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:2447]: Stray space at the end here

File: src/locale/locales/ca/messages.po [hunk 6]
```diff
@@ -2986,8 +2985,8 @@ msgstr "Cerca"
 
 #: src/view/screens/Search/Search.tsx:628
 #: src/view/shell/desktop/Search.tsx:255
-msgid "Search for \"{query}\""
-msgstr ""
+msgid "Search for \"{query}\
+msgstr "Cerca per \"{query}\
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:2989]: Missing quotation marks here

File: src/locale/locales/ca/messages.po [hunk 7]
```diff
@@ -3584,11 +3583,11 @@ msgstr "Aquest canal està rebent moltes visites actualment i està temporalment
 #: src/view/screens/ProfileFeed.tsx:484
 #: src/view/screens/ProfileList.tsx:639
 msgid "This feed is empty!"
-msgstr ""
+msgstr "Aquest canal està buit!"
 
 #: src/view/com/posts/CustomFeedEmptyState.tsx:37
 msgid "This feed is empty! You may need to follow more users or tune your language settings."
-msgstr ""
+msgstr Aquest canal està buit! Necessites seguir més usuaris o modificar la teva configuració d'idiomes"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:3590]: Missing opening quotation mark here

File: src/locale/locales/ca/messages.po [hunk 8]
```diff
@@ -3856,12 +3855,12 @@ msgid "users followed by <0/>"
 msgstr "usuaris seguits per <0/>"
 
 #: src/view/com/modals/Threadgate.tsx:106
-msgid "Users in \"{0}\""
+msgid "Users in \"{0}\
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:3858]: Missing quotation marks here I think? Not sure if it matters

File: src/locale/locales/ca/messages.po [hunk 9]
```diff
@@ -111,35 +111,35 @@ msgstr "Compte"
...
 msgid "Account muted"
-msgstr ""
+msgstr "Compte silenciat"
 
 #: src/view/com/modals/ModerationDetails.tsx:86
 msgid "Account Muted"
-msgstr ""
+msgstr "Compte Silenciat"
 
 #: src/view/com/modals/ModerationDetails.tsx:72
```
Comment (jordimas) [inline src/locale/locales/ca/messages.po:124]: La S de "Silenciat" van en minúscules "silenciat"

File: src/locale/locales/ca/messages.po [hunk 10]
```diff
@@ -1039,31 +1039,31 @@ msgstr "Fet{extraText}"
...
 msgid "e.g. The posters who never miss."
-msgstr ""
+msgstr "p.e. Els que mai fallen"
 
 #: src/view/com/modals/CreateOrEditList.tsx:245
 msgid "e.g. Users that repeatedly reply with ads."
-msgstr ""
+msgstr "p.e. Usuaris que sempre responen amb anuncis"
 
 #: src/view/com/modals/InviteCodes.tsx:96
```
Comment (jordimas) [inline src/locale/locales/ca/messages.po:1068]: En català "p.e." s'escriu "p. ex." (amb espai). Aplica a les tres frases anteriors.

File: src/locale/locales/ca/messages.po [hunk 11]
```diff
@@ -4028,11 +4027,11 @@ msgstr "Has bloquejat l'autor o has estat bloquejat per ell."
 
 #: src/view/com/modals/ModerationDetails.tsx:56
 msgid "You have blocked this user. You cannot view their content."
-msgstr ""
+msgstr "Has bloquejat aquest usuari. No pots veure el seu contingut."
 
 #: src/view/com/modals/ModerationDetails.tsx:87
 msgid "You have muted this user."
-msgstr ""
+msgstr "Has silenciat aquest usuari"
```
Comment (jordimas) [inline src/locale/locales/ca/messages.po:4035]: Falta el punt al final.

File: src/locale/locales/ca/messages.po [hunk 12]
```diff
@@ -3910,23 +3909,23 @@ msgstr "Visita el lloc web"
 
 #: src/view/com/modals/ContentFilteringSettings.tsx:254
 msgid "Warn"
-msgstr ""
+msgstr "Arverteix"
```
Comment (jordimas) [inline src/locale/locales/ca/messages.po:3913]: Arverteix -> Adverteix

File: src/locale/locales/ca/messages.po (hunk 8 above)
Comment (ivanbea) [reply src/locale/locales/ca/messages.po:3858]: Thank you, you are really helpful

Comment (surfdude29) [reply src/locale/locales/ca/messages.po]: Happy to help :)

**Comment by pfrazee** [comment]
Good to go?

**Comment by ivanbea** [comment]
Yes, I corrected the errors found by @surfdude29 and @jordimas and it looks ready to go now

**Comment by MiquelAdell** [comment]
Were the lines not previously appearing on the po file? They were introduced in the last update or something like that?
If that is the case the file looks fine to me.

**Comment by ivanbea** [comment]
Yes, they added more lines for localization, and I suspect that this will not be the last update

**Review by darccio — COMMENTED** [review]
LGTM

**Comment by pfrazee** [comment]
Awesome, thanks everyone! Merged

**Pull request merged**